test/services/notifications/setup/fetch-returns-due.service.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, before } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const ReturnLogHelper = require('../../../support/helpers/return-log.helper.js')
+
+// Thing under test
+const FetchReturnsDueService = require('../../../../app/services/notifications/setup/fetch-returns-due.service.js')
+
+describe('Notifications Setup - Fetch returns due service', () => {
+  let dueDate
+  let isSummer
+  let licenceRefs
+  let returnLog
+
+  before(async () => {
+    dueDate = '2024-04-28' // This needs to differ from any other returns log tests
+    isSummer = 'false'
+
+    returnLog = await ReturnLogHelper.add({})
+
+    // Add an additional returns log
+    await ReturnLogHelper.add({})
+
+    dueDate = returnLog.dueDate
+
+    licenceRefs = [returnLog.licenceRef]
+  })
+
+  describe('when there are licences', () => {
+    it('correctly returns the matching licences', async () => {
+      const result = await FetchReturnsDueService.go(licenceRefs, dueDate, isSummer)
```

Comment:
We have a `dueDate` variable which we set in the `beforeEach()`. But we don't then use the value when creating the return logs.

Instead, we overwrite what we set with whatever the helper used when creating the return  log. And, though I know the helper will use the same 'due date', it's not clear from the test that the second return log will have the same values as the first.

I'd suggest passing in `dueDate` and `isSummer` to both calls to `ReturnLogHelper.add({})` so it is clearer the difference between them is the licence ref (might help to specify those as well? 🤔 )

N.B. If you don't need to override any defaults, you don't need to pass an empty object in.